csm/conf/prepare.py

```diff
@@ -105,7 +105,7 @@ def _set_fqdn_for_nodeid(self):
             node_name = node_data.get(const.NAME)
             Conf.set(const.CSM_GLOBAL_INDEX, f"{const.MAINTENANCE}>{node_name}",f"{hostname}")
 
-    def _get_data_nw_info(self):
+    def _get_consul_info(self):
         """
         Obtains minion data network info.
 
```

Comment:
Minion id is not the part params .. remove it from  method doc.. Please add the description of return value in method doc..  Write proper description.

Reply:
Updated required changes in by adding proper description.